design/software-architecture/index.qmd

```diff
@@ -1,5 +1,73 @@
 ---
-title: "Overview of report"
+title: "Software architecture documentation"
+order: 0
 ---
 
-<!-- Following [arc42](https://arc42.org) and [C4](https://c4model.com/) -->
+{{< include ../../includes/_wip.qmd >}}
+
+## Purpose
+
+The core purpose of writing these architecture documentation is to be
+able to describe to ourselves and others what Seedcase is aiming to
+accomplish and how it will do that.
+
+We structured the software architecture documentation to follow the
+[arc42](https://arc42.org) template and the [C4
+Model](https://c4model.com/) as much as is relevant for our situation,
+since these both are fairly popular and common templates to use. These
+documentation are also continuously evolving documentation as we learn,
+test, develop, and grow as a team.
+
+## Reader
+
+The assumed readers are similar to the ones we listed in the [Design of
+Seedcase](../index.md) landing page. The complete list of expected
+readers is found in the [Stakeholders](introduction.qmd#stakeholders)
+section of the Introduction.
+
+## Section overview
+
+1.  [Introduction and goals](introduction.qmd): Targeted to any reader.
```

Comment:
Is it targeted at or targeted to?  Personally I prefer 'at' but I believe both are correct.